public/components/cloud/package.go

```diff
@@ -24,13 +24,16 @@ import (
 	_ "github.com/redpanda-data/connect/v4/public/components/changelog"
 	_ "github.com/redpanda-data/connect/v4/public/components/confluent"
 	_ "github.com/redpanda-data/connect/v4/public/components/crypto"
+	_ "github.com/redpanda-data/connect/v4/public/components/dgraph"
 	_ "github.com/redpanda-data/connect/v4/public/components/gcp/enterprise"
 	_ "github.com/redpanda-data/connect/v4/public/components/io"
 	_ "github.com/redpanda-data/connect/v4/public/components/kafka"
+	_ "github.com/redpanda-data/connect/v4/public/components/kafka/enterprise"
 	_ "github.com/redpanda-data/connect/v4/public/components/maxmind"
 	_ "github.com/redpanda-data/connect/v4/public/components/memcached"
 	_ "github.com/redpanda-data/connect/v4/public/components/msgpack"
 	_ "github.com/redpanda-data/connect/v4/public/components/nats"
+	_ "github.com/redpanda-data/connect/v4/public/components/ollama"
```

Comment:
We don't want this in cloud - only in the special AI container

Reply:
just realised, will fix